packages/parser-package-json/src/schema.ts

```diff
@@ -0,0 +1,308 @@
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
```

Comment:
Very nice! I played around with this a bit in VS Code and really liked getting the intellisense suggestions when accessing the config object.

Since this file is auto-generated, let's add a (pre-)build step to generate it from the current `schema.json` and a local `.gitignore` to exclude the generated file from source control.

Reply:
The auto-generated schema did have an error which I had to fix by hand (against orders) -(scripts prop on generated schema for package.json needs to be fixed)[https://github.com/bcherny/json-schema-to-typescript/issues/210]  ... not sure how to proceed ...

Reply:
Looks like editing the schema.json causes the ajv validator to  not work as expected, resulting in valid `package.json` configurations of fail the tests. I've open an issue to track the auto-generation of the TS schema from JSON schema at [Add pre-build step to auto-generate Typescript schema from JSON schema #1594](https://github.com/webhintio/hint/issues/1594)

Reply:
@antross I guess, that was the only thing we were missing to close the PR, right?